includes/wf_crm_webform_preprocess.inc

```diff
@@ -843,7 +847,7 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
       if (is_array($value)) {
         $value = implode(', ', $value);
       }
-      $values[] = implode(' &amp; ', wf_crm_explode_multivalue_str(check_plain($value)));
+      $values[] = implode(' &amp; ', wf_crm_explode_multivalue_str(Html::escape($value)));
```

Comment:
Given all the Twig autoescape stuff that happens in D8, I'd worry that this would lead to double escaping. Should this maybe just plain strings and rely on the Twig automagic?

Reply:
You're right.

* https://www.drupal.org/project/drupal/issues/2089331
* https://www.drupal.org/node/2549395

Deprecation message;

```
Will be removed before Drupal 9.0.0. Rely on Twig's auto-escaping feature, or use the #plain_text key when constructing a render array that contains plain text in order to use the renderer's auto-escaping feature. If neither of these are possible, \Drupal\Component\Utility\Html::escape() can be used in places where explicit escaping is needed.
```